TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/human/HumanSocketIOTpService.java

```diff
@@ -0,0 +1,117 @@
+package com.novoda.tpbot.human;
+
+import android.os.Handler;
+import android.os.Looper;
+
+import com.novoda.tpbot.Direction;
+import com.novoda.tpbot.Result;
+import com.novoda.tpbot.support.Observable;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+import io.socket.client.Ack;
+import io.socket.client.IO;
+import io.socket.client.Socket;
+import io.socket.client.SocketIOException;
+import io.socket.emitter.Emitter;
+
+import static io.socket.client.Socket.EVENT_CONNECT_ERROR;
+import static io.socket.client.Socket.Listener;
+
+class HumanSocketIOTpService implements HumanTpService {
```

Comment:
Could be, I guess the package will be enough to show that this is for the human part. 👍